src/CLI/actioner/sonic-cli-show-techsupport.py

```diff
@@ -22,42 +23,47 @@ def invoke(func, args):
 
 def run(func, args):
 
+    if func != 'rpc_sonic_show_techsupport_sonic_show_techsupport_info':
+        print("ERROR: Python: Show Techsupport parsing Failed: Invalid "
```

Comment:
Let's get consistent and have all error messages prefixed with "% Error:"
Let's not have internal information (ex: python)

Reply:
I have modified this error message (and all other 'show tech-support" error messages to match the "%Error: ..." format.